doc/man3/SSL_CTX_set1_curves.pod

```diff
@@ -174,6 +174,33 @@ and the client supports C<P-384> including key share for this group, a HRR will
 triggered for C<P-521> despite the availability of a key share for P-384, which
 overlaps with a lower priority server-side tuple.
 
+For example, consider the same server but different client with I<list>
```

Comment:
Please consider changing the first sentence on line 115 to:
`Group tuples of comparable security are defined by separating them from each other by a tuple separator C</>.`